ingest-app/src/cmr/ingest/api/ingest.clj

```diff
@@ -12,7 +15,7 @@
    [cmr.common.services.errors :as srvc-errors]
    [cmr.common.util :as util]
    [cmr.common.xml.gen :refer :all]
-   [cmr.common-app.api.enabled :as common-enabled]
+   [cmr.ingest.services.bulk-update-service :as bulk-update]
```

Comment:
Is this only ever going to be an update service? Or are we going to try other bulk operations?